articles/121_realtime_proposal.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: default
+title: Proposal for Implementation of Real-time Systems in ROS 2
+permalink: articles/realtime_proposal.html
+abstract: Proposal for a test-driven approach to the real-time performance requirement in ROS 2.
+published: true 
+author: Jackie Kay
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Requirements and Implementation of Real-Time Systems
+
+### System requirements
+
+The requirements of a real-time system vary depending on the use case.
+At its core, the real-time requirement of a system has two components:
+
+- Latency
+
+  - Update period
```

Comment:
I'm probably being pedantic, but for consistency with the "failure mode" comment, perhaps include the term "deadline" here, too.